src/scripts/start_instance.js

```diff
@@ -4,8 +4,10 @@ const topic = process.argv[2];
 
 (async () => {
   const runner = steveo.runner();
-  if (runner.registry.getTask(topic)) {
-    await runner.process([topic]);
+  while (true) {
```

Comment:
@arshu16 with the mp-example, I was seeing this get respawned rather than being a long lived process, was that intended. I added the while(true)